pincer/objects/guild/overwrite.py

```diff
@@ -31,3 +32,8 @@ class Overwrite(APIObject):
     type: int
     allow: str
     deny: str
+
+    @property
+    def permissions(self) -> Permission:
```

Comment:
the name's permission__s__ yet the return type is `Permission`, sounds a bit confusing imo